payloads/library/Notepad Fun/payload.txt

```diff
@@ -1,34 +1,72 @@
-#
-#
-# Title:             Notepad Fun
-# Author:            The10FpsGuy and Mrhut10
-#                    Big thanks to Mrhut10 for helping to put a loop in it :D
-# Category:          Fun
-#
-#
-#options
-LED B G 100
-msg_header="begining"
-msg_body="I will learn to lock my computer"
-msg_body_repeats=10
-msg_end="Please remember to lock your computer when you walk away."
-
-ATTACKMODE HID
-Q GUI r
-Q DELEY 150
-Q STRING notepad.exe
-Q ENTER
-LED B 500
-
-Q DELAY 200
-Q STRING $msg_header
-Q ENTER
-
-for (( i=1; i<=$msg_body_repeats; i++ ))
-do
-   Q STRING $msg_body
-   Q ENTER
-done
-Q STRING $msg_end
-Q ENTER
-LED G 0
+
+#!/bin/bash
+#
+# Title:             Notepad Fun
+# Author:            The10FpsGuy and Mrhut10
+#                    Big thanks to Mrhut10 for helping to put a loop in it :D
+#                    Edits by Biocow to create separate header, body and footer files.
+# Target Platform:   Windows
+# Category:          Fun
+#
+# DESCRIPTION:
+# Opens Notepad on Windows Systems and leaves a message for the owner of the machine
+#
+# Requires Firmware 1.1
+#
+
+# SETUP
+# Select Files
+LED SETUP
+
+header_file=header.txt
+  body_file=body.txt
+footer_file=footer.txt
+
+# How many times do you want each file to repeat? (Zero is a valid entry)
+header_repeats=1
+body_repeats=5
+footer_repeats=1
+
+# #############################
+# DO NOT EDIT BELOW THIS LINE #
+# #############################
+
+# Attack Setup
+ATTACKMODE STORAGE HID VID_0X05AC PID_0X021E
+source bunny_helpers.sh
```

Comment:
bunny_helpers is not part of the 1.1 firmware. the new way in 1.1 is to simply call the 'get' funtion GET SWITCH_POSITION

source bunny_helpers.sh can be commented out ( unless for backwards compat )